lib/LedgerSMB/Upgrade_Tests.pm

```diff
@@ -24,6 +24,18 @@ use LedgerSMB::Locale qw(marktext);
 
 =over
 
+=item get_presets()
+
+Returns the test array
```

Comment:
Which test array? (Note, this is a request for more documentation, not for an answer to the question in a GitHub response)